manifests/g/GiulioSorrentino/Backgrounds/1.0/GiulioSorrentino.Backgrounds.locale.it-IT.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GiulioSorrentino.Backgrounds
+PackageVersion: 1.0
+PackageLocale: it-IT
+Publisher: Giulio Sorrentino
+PublisherUrl: http://numerone.altervista.org
+Author: Giulio Sorrentino
+PackageName: numeronesoft backgrounds
+PackageUrl: http://numerone.altervista.org/Downloads/numeronesoft-backgrounds.exe

```

Comment:
Does this package have its own website or page? If not, we should remove this field.